modules/ROOT/pages/04-scaling.adoc

```diff
@@ -282,7 +292,7 @@ kubectl::
 [#run-scaling-cleanup]
 [source,bash,subs="attributes+,+macros"]
 ----
-kubectl -n {tutorial-namespace} delete services.serving.knative.dev greeter  &&\
+kubectl -n {tutorial-namespace} delete services.serving.knative.dev greeter
```

Comment:
If the first command fails (and it does if you just follow the tutorial flow), deletion of `prime-generator` fails.

I changed this 1-liner with a 2-liner but using an `||` is also good.